interfaces/odata.js

```diff
@@ -19,8 +19,8 @@ declare module 'odata' {
   };
 
   declare type OHandler<TModel> = {
-     inlineCount: number,
```

Comment:
eh.. o.js has `count` and `inlineCount` methods and `inlinecount` var. so I can't rename `inlinecount` to `count`